CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 - [#187](https://github.com/smile-io/ember-polaris/pull/187) [FEATURE] Add `disabled` attribute to `polaris-choice`
 - [#185](https://github.com/smile-io/ember-polaris/pull/185) [FEATURE] Update `polaris-button` to accept components for `icon` value
 - [#191](https://github.com/smile-io/ember-polaris/pull/191) [ENHANCEMENT] Update `polaris-layout/annotated-section` styling to match Polaris v2.11.0
-
+- [#193](https://github.com/smile-io/ember-polaris/pull/193) [FEATURE] Update `polaris-choice-list` to render `error` and `aria` attributes
```

Comment:
this is not a new feature 😜...update? (a few others here are marked as feature when they shouldn't, ex: button)